package.json

```diff
@@ -17,6 +17,7 @@
     "@tailwindcss/typography": "^0.5.9",
     "@vercel/analytics": "^1.0.1",
     "autoprefixer": "^10.4.14",
+    "date-fns": "^2.30.0",
```

Comment:
We have our own project date helper `services/utils/dateFormat.js` because we reached the limit of Vercel files - so we have to be very careful and selective of what libraries we include